File: Front-end/src/app/shared/auditEntity.ts
import { BaseEntity } from '../shared/baseentity';
export interface AuditEntity extends BaseEntity {
		createdBy?: number;
		createdOn: Date;
		modifiedBy?: number;
		modifiedOn?: Date;
		isDeleted: boolean;
		deletedBy?: number;
		deletedOn?: Date;
}

File: Service/Z2data.Core/AuditEntity.cs
using System;

namespace Z2data.Core
{
    public abstract partial class AuditEntity: BaseEntity
    {
        public int? CreatedBy { get; set; }
        public System.DateTime CreatedOn { get; set; }
        public int? ModifiedBy { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public bool IsDeleted { get; set; }
        public int? DeletedBy { get; set; }
        public DateTime? DeletedOn { get; set; }
    }
}